apps/member/src/components/group/AssignmentUploadSection/AssignmentUploadSection.tsx

```diff
@@ -152,21 +158,28 @@ const AssignmentUploadSection = ({
               placeholder={description || '제출물 설명을 입력해주세요.'}
               value={description}
               onChange={handleDescriptionChange}
+              disabled={!editMode && !(myAssignment === undefined)} // 제출물이 없는 상태엔 입력 가능, 제출물이 있지만 수정하기 버튼이 클릭되지 않으면 수정 불가
             />
           </Table.Cell>
         </Table.Row>
       </Table>
       <div className="mt-2 flex gap-4">
-        {uploadedFile && (
+        {uploadedFile !== null && !editMode && (
```

Comment:
해당 경우도 아래처럼 작성하는게 더 깔끔해보일거같아요

```suggestion
        {uploadedFile && !editMode && (
```